pkg/quarterdeck/db/models/users.go

```diff
@@ -320,6 +321,14 @@ const (
 // empty (nil) slice if there are no results. If there is a next page of results, e.g.
 // there is another row after the page returned, then a cursor will be returned to
 // compute the next page token with.
+
+// Query construction with pageSize only:
+// SELECT id, name, email, terms_agreement, privacy_agreement, last_login, created, modified from users
+// where id in (select user_id FROM organization_users WHERE organization_id=:orgID) LIMIT :pageSize
+//
+// Query construction with pageSize and endIndex:
+// SELECT id, name, email, terms_agreement, privacy_agreement, last_login, created, modified from users
+// where id in (select user_id FROM organization_users WHERE organization_id=:orgID) AND id > :endIndex LIMIT :pageSize
```

Comment:
Thanks for adding the queries, that definitely clears things up a bit. I was wondering if we should move this down into the body of the method since it describes the _implementation_. As a programmer looking at the method in an editor, I am usually more interested in the _interface_ (e.g. what do I have to pass in to make this work).

Reply:
that makes sense - I have made the change